src/main/java/org/openrewrite/java/migrate/apache/commons/io/ApacheCommonsFileUtils.java

```diff
@@ -47,5 +48,30 @@ File after(String name) {
 //        }
 //    }
 
+    private static class CopyDirectory {
+        @BeforeTemplate
+        void before(File a, File b) throws Exception {
+            FileUtils.copyDirectory(a, b);
+        }
+
+        @AfterTemplate
+        void after(File a, File b) throws Exception {
+            Files.copy(a.toPath(), b.toPath());
+        }
+    }
+
+    @SuppressWarnings("deprecation")
+    private static class WriteStringToFile {
+        @BeforeTemplate
+        void before(File a, String s) throws Exception {
+            FileUtils.writeStringToFile(a, s);
+        }
+
+        @AfterTemplate
+        void after(File a, String s) throws Exception {
+            Files.write(a.toPath(), s.getBytes());
```

Comment:
This looks good, AFAICT. We could also add additional recipes to cover some of the other signatures (like `writeStringToFile(File, String, boolean)`, `writeStringToFile(File, String, Charset)`, and `writeStringToFile(File, String, Charset, boolean)`). But it is probably worth doing a search in the SaaS to check how often these get used. Probably they could all be covered by the same recipe, just using different before templates (all with the same input signatures).

Reply:
Sounds good, I'll check the SaaS and see how often these overloads get used. I wanted to make sure this first template was accurate before worrying about overloads.

Reply:
Sounds like we could use a few more overloads here and merge those separate from the above `CopyDirectory`. Would you mind finishing this one @AlekSimpson ?

Reply:
Would also be good to add a single document example test that shows which variants are replaced.